src/Components/ScrivitoExtensions/ChatbotTab/model.js

```diff
@@ -1,7 +1,7 @@
 export function getModel() {
   const fallback = "gpt-4o-mini";
   try {
-    return localStorage.getItem("openai_model") || fallback;
+    return localStorage.getItem("openai_model").trim() || fallback;
```

Comment:
* [ ] FYI This will create an exception control flow for virtually all users for no good reason.